src/main/java/com/minecolonies/coremod/event/EventHandler.java

```diff
@@ -190,58 +194,77 @@ public void onEnteringChunk(@NotNull final PlayerEvent.EnteringChunk event)
         if (entity instanceof EntityPlayerMP)
         {
             final World world = entity.getEntityWorld();
-            MineColonies.getNetwork().sendTo(new UpdateChunkRangeCapabilityMessage(world, event.getNewChunkX(), event.getNewChunkZ(), Configurations.gameplay.workingRangeTownHallChunks), (EntityPlayerMP) event.getEntity());
+            MineColonies.getNetwork()
+              .sendTo(new UpdateChunkRangeCapabilityMessage(world, event.getNewChunkX(), event.getNewChunkZ(), Configurations.gameplay.workingRangeTownHallChunks),
+                (EntityPlayerMP) event.getEntity());
 
             final Chunk newChunk = world.getChunk(event.getNewChunkX(), event.getNewChunkZ());
             ChunkDataHelper.loadChunk(newChunk, entity.world);
 
             final IColonyTagCapability newCloseColonies = newChunk.getCapability(CLOSE_COLONY_CAP, null);
 
-            MineColonies.getNetwork().sendToAll(new UpdateChunkCapabilityMessage(newCloseColonies, newChunk.x, newChunk.z));
+            MineColonies.getNetwork().sendTo(new UpdateChunkCapabilityMessage(newCloseColonies, newChunk.x, newChunk.z), (EntityPlayerMP) entity);
```

Comment:
For whoever reviews this, this was a bug, there was no need to send this message to so many people